import_processor_async/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+This module does add an asynchronous mode to the Import Processor module so that the user
+does not have to wait for data imports to succeed.
```

Comment:
Please add the note that the pre-processor and post-processor are disabled in this mode.